test/automl/test_split.py

```diff
@@ -80,6 +80,19 @@ def test_groups():
     automl_settings["eval_method"] = "holdout"
     automl.fit(X, y, **automl_settings)
 
+    automl_settings["split_type"] = GroupKFold(n_splits=3)
+    try:
+        automl.fit(X, y, **automl_settings)
+        raise RuntimeError(
+            "GroupKFold object as split_type should fail when eval_method is holdout"
+        )
+    except AssertionError:
```

Comment:
Why catching AssertationError?

Reply:
This is test code. We expect AssertionError to be raised in this case.